Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/TelemetryConstants.java

```diff
@@ -102,6 +103,15 @@
     public static final String SHOWPROP_SPRING_CLOUD_APP = "showprop-springcloud-app";
     public static final String OPEN_IN_BROWSER_SPRING_CLOUD_APP = "open-inbrowser-springcloud-app";
     public static final String OPEN_IN_PORTAL_SPRING_CLOUD_APP = "open-inportal-springcloud-app";
+    public static final String MYSQL_CREATE = "mysql-create";
+    public static final String MYSQL_DELETE = "mysql-delete";
+    public static final String MYSQL_START = "mysql-start";
+    public static final String MYSQL_STOP = "mysql-stop";
+    public static final String MYSQL_RESTART = "mysql-restart";
+    public static final String MYSQL_OPEN_IN_PORTAL = "mysql-open-in-portal";
+    public static final String MYSQL_SHOW_PROPERTIES = "mysql-show-properties";
+    public static final String MYSQL_CONNECT_TO_SERVER = "mysql-connect-to-server";
+    public static final String MYSQL_BIND_INTO = "mysql-bind-into";
```

Comment:
bind-into -> bind-project?

Reply:
will remove it in my next PR. it is unnecessary currently.